x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/overview/overview.tsx

```diff
@@ -44,18 +65,45 @@ export const Overview = () => {
     />
   );
 
+  const shouldShowCallout = () => {
+    if (dataStreamsStatus !== 'success') {
+      return false;
+    }
+
+    const { type } = asset;
+    const baseCondition =
+      !isMetricsSignal(dataStreams) && !dismissedAddMetricsCallout && renderMode.mode === 'page';
```

Comment:
I'd like to simplify this... It looks we need to run so many checks just to show the call out.

```suggestion
    if (dataStreamsStatus !== 'success' 
       || renderMode.mode !== 'page'
       || dismissedAddMetricsCallout) {
      return false;
    }

    const { type } = asset;
    const baseCondition = !isMetricsSignal(dataStreams);
```